root/static/scripts/edit/utility/guessFeat.js

```diff
@@ -10,24 +10,59 @@ const {MIN_NAME_SIMILARITY} = require('../../common/constants');
 const {artistCreditFromArray} = require('../../common/immutable-entities');
 const MB = require('../../common/MB');
 const clean = require('../../common/utility/clean');
+const {
+    fromFullwidthLatin,
+    hasFullwidthLatin,
+    toFullwidthLatin
+} = require('./fullwidthLatin');
 const getSimilarity = require('./similarity');
 
-var featRegex = /(?:^\s*|[,\-]\s*|\s+)(?:(?:ft|feat)[.\s]|featuring\s+)/i;
-var collabRegex = /(,?\s+(?:&|and|et)\s+|,\s+|;\s+|\s*\/\s*|\s+vs\.\s+)/i;
-var bracketPairs = [['(', ')'], ['[', ']']];
+var featRegex = /(?:^\s*|[,，－\-]\s*|\s+)((?:ft|feat|ｆｔ|ｆｅａｔ)(?:[.．]|(?=\s))|(?:featuring|ｆｅａｔｕｒｉｎｇ)(?=\s))\s*/i;
+var collabRegex = /([,，]?\s+(?:&|and|et|＆|ａｎｄ|ｅｔ)\s+|[,，;；]\s+|\s*[\/／]\s*|\s+(?:vs|ｖｓ)[.．]\s+)/i;
+var bracketPairs = [['(', ')'], ['[', ']'], ['（', '）'], ['［', '］']];
```

Comment:
Can you add a small test for the change to this line? I reverted it and the tests still pass, so I'm not sure what's it's doing.

Reply:
Added one for brackets and one for collaboration in https://github.com/metabrainz/musicbrainz-server/pull/697/commits/266d45c4693e5bda12004427c709ad97d796aea2.

Also updated code in https://github.com/metabrainz/musicbrainz-server/pull/697/commits/eec231df62a907234d05641ba8f5caa2a09be59c and tests in https://github.com/metabrainz/musicbrainz-server/pull/697/commits/4cef2c60d54ceedd6dd0f6e9685736f100d5eee9 to fully convert collaboration join phrase to fullwidth Latin if one such character existed, for consistency with the behaviour for featuring.